libraries/chain/include/eosio/chain/asset.hpp

```diff
@@ -14,63 +15,64 @@
 
 namespace eosio { namespace chain {
 
-using asset_symbol = uint64_t;
+      //using asset_symbol = uint64_t;
 
 struct asset
```

Comment:
This would be a great time to add doxygen.

Reply:
I included comments and will add more.